lib/internal/child_process.js

```diff
@@ -230,12 +230,12 @@ util.inherits(ChildProcess, EventEmitter);
 
 
 function flushStdio(subprocess) {
-  if (subprocess.stdio == null) return;
-  subprocess.stdio.forEach(function(stream, fd, stdio) {
-    if (!stream || !stream.readable || stream._readableState.readableListening)
-      return;
-    stream.resume();
-  });
+  if (subprocess.stdio === null || subprocess.stdio === undefined) return;
```

Comment:
Motivation of this change was [this line](https://github.com/notarseniy/node/blob/bbbabaf8730bad0e73f196e3d63195f97969a98a/lib/internal/child_process.js#L799) with that-'style' condition.